### PR TITLE
feat(timed): add buildProject value for backend

### PR DESCRIPTION
**Pull request by trowik**
# Description

add buildProject value for backend

# Issues

# Checklist

* [x] This PR contains a description of the changes I'm making
* [x] I updated the version in Chart.yaml
* [x] I updated the changelog with an `artifacthub.io/changes` annotation in `Chart.yaml`, check the [example](docs/development.md#Changelog) in the documentation.
* [x] I updated applicable README.md files using  `pre-commit run`
* [x] I documented any high-level concepts I'm introducing in `docs/`
* [x] CI is currently green and this is ready for review
* [x] I am ready to test changes after they are applied and released

### CODE REVIEW

**Comment by trowik** [comment]
not sure about the chart version

File: charts/timed/values.yaml [hunk 1]
```diff
@@ -104,6 +104,8 @@ backend:
     admins: []
     # -- Define allowed domains for CORS
     corsAllowedOrigins: []
+    # -- Define name for build projects
+    buildProject: " BUILD"
```
Comment (hairmare) [inline charts/timed/values.yaml:108]: is this whitespace legit?
```suggestion
    buildProject: "BUILD"
```
what's a build project?

**Comment by hairmare** [comment]
you might also need to add something in `charts/timed/templates/configmap-backend.yaml` for this flag to do it's thing

File: charts/timed/values.yaml (hunk 1 above)
Comment (trowik) [reply charts/timed/values.yaml:108]: The space there is on purpose.

Not really sure, but it's part of the Plan-Build-Run Model.
As far as I know we define them by using different cost centers in Timed , e.g. "DEVELOPMENT BUILD"

**Comment by hairmare** [comment]
> not sure about the chart version

it's semver, so 0.x means you can do whatever you want. we tend to mostly increment the minor and only really update the patch release for actual fixes. at some point when charts start reaching 1.x, we'll have some real rules...

https://semver.org/

**Comment by trowik** [comment]
> you might also need to add something in `charts/timed/templates/configmap-backend.yaml` for this flag to do it's thing

The value is just an env var that I already added to `config-backend.yaml`

**Comment by trowik** [comment]
> > not sure about the chart version
> 
> it's semver, so 0.x means you can do whatever you want. we tend to mostly increment the minor and only really update the patch release for actual fixes. at some point when charts start reaching 1.x, we'll have some real rules...
> 
> https://semver.org/

alright, so 0.16.0 then

File: charts/timed/values.yaml (hunk 1 above)
Comment (hairmare) [reply charts/timed/values.yaml:108]: oh, so the commercial "BUILD" and not something technical :sweat_smile: 

Do you know if maybe the existing `.redmine.buildProject` should be merged into this variable? 

https://github.com/adfinis/helm-charts/blob/dd9d46a7907c92f46615e018bdeee1f562410de0/charts/timed/templates/configmap-backend.yaml#L77

Comment (trowik) [reply charts/timed/values.yaml:108]: that one defines the name in Redmine, that is different from the one used in Timed

**Review by hairmare — APPROVED** [review]

**Comment by trowik** [comment]
forgot to change the chart version...

**Comment by hairmare** [comment]
> forgot to change the chart version...

doesn't matter that much, that's the nice thing about 0.x :)

**Pull request merged**